husky_directory/models/search.py

```diff
@@ -28,19 +27,24 @@ class SearchDirectoryInput(DirectoryBaseModel):
     phone: Optional[str] = Field(None, regex=r"^[0-9]+$", max_length=32)
```

Comment:
Line 27 and 43, what the difference between phone and phone contacts?
Also, is phone optional but phone contacts is not? Sorry just need some clarification.

Reply:
`phone` should have been removed.

Reply:
I believe you review this again you'll find this is already taken care of. I had a bit of a rebasing issue.